source/mongodb-handshake/handshake.rst

```diff
@@ -3,7 +3,7 @@ MongoDB Handshake
 =================
 
 :Spec Title: MongoDB Handshake
-:Spec Version: 1.1.0
+:Spec Version: 1.1.1
```

Comment:
As this is is a new feature, make it a minor version bump, not a patch bump.

Reply:
Done